src/sentry/models/debugfile.py

```diff
@@ -395,9 +397,10 @@ def detect_dif_from_path(path, name=None, debug_id=None):
         try:
             BcSymbolMap.open(path)
         except SymbolicError as e:
-            logger.warning("bcsymbolmap.bad-file", exc_info=True)
+            logger.debug(f"File failed to load as BCSymbolmap: {path}")
             raise BadDif("Invalid BCSymbolMap: %s" % e)
         else:
+            logger.debug(f"File loaded as BCSymbolMap: {path}")
```

Comment:
would never use fstrings or any generic string interpolation with logging, there's value in having more structured log records where the message is separate from parameters

Reply:
oops, yes.  also no point in formatting debug-messages on prod.